### PR TITLE
Move TopNav to SideNav

### PR DESCRIPTION
**Pull request by gettinToasty**

### CODE REVIEW

File: test/themes.ts [hunk 1]
```diff
@@ -15,8 +15,8 @@ test('Installing a theme', async (t: any) => {
 
   await logIn(t);
 
-  await app.client.waitForExist('.top-nav.loading', 5000, true);
-  await app.client.click('button=Themes');
+  // await app.client.waitForExist('button=themes', 5000, true);
```
Comment (holiber) [inline test/themes.ts]: Seems like it's gonna be flaky

File: test/undo.ts [hunk 2]
```diff
@@ -14,11 +14,15 @@ import { sleep } from './helpers/sleep';
 useSpectron();
 
 async function undo(t: TExecutionContext) {
-  await t.context.app.client.click('.fa-undo');
+  t.context.app.client.keys(['Control', 'z']);
+  t.context.app.client.keys('Control');
+  await sleep(500);
```
Comment (holiber) [inline test/undo.ts]: add await for sending keys instead of sleep
```
await t.context.app.client.keys(['Control', 'z']);
```

File: test/undo.ts [hunk 3]
```diff
@@ -14,11 +14,15 @@ import { sleep } from './helpers/sleep';
...
 }
 
 async function redo(t: TExecutionContext) {
-  await t.context.app.client.click('.fa-redo');
+  t.context.app.client.keys(['Control', 'y']);
+  t.context.app.client.keys('Control');
+  await sleep(500);
```
Comment (holiber) [inline test/undo.ts]: same as above

File: test/themes.ts (hunk 1 above)
Comment (gettinToasty) [reply test/themes.ts]: how so?

File: test/undo.ts (hunk 2 above)
Comment (gettinToasty) [reply test/undo.ts]: keys doesn't return a promise it cant be awaited

File: test/themes.ts (hunk 1 above)
Comment (holiber) [reply test/themes.ts]: Just reviewed the code. We have updated `logIn(t)` method now. So this is not gonna be an issue.

File: test/undo.ts (hunk 2 above)
Comment (holiber) [reply test/undo.ts]: Just looked into the source code. It returns promise but seems like d.ts files are incorrect.
Try to use a typecasting here
```
await ((this.client.keys(['Control', 'z']) as any) as Promise<any>); 
```

**Pull request merged**